source/tests/pd/model/test_descriptor_dpa1.py

```
@@ -0,0 +1,387 @@
+        )
+        with open(Path(CUR_DIR) / "models" / "dpa1.json") as fp:
+            self.model_json = json.load(fp)
+        self.file_model_param = Path(CUR_DIR) / "models" / "dpa1.pd"
+        self.file_type_embed = Path(CUR_DIR) / "models" / "dpa2_tebd.pd"
+
+    def test_descriptor_block(self) -> None:
+        # paddle.seed(0)
+        model_dpa1 = self.model_json
+        dparams = model_dpa1["descriptor"]
+        ntypes = len(model_dpa1["type_map"])
+        assert "se_atten" == dparams["type"]
```

Ruff (0.8.0)

Yoda condition detected

Rewrite as `dparams["type"] == "se_atten"`

(SIM300)